src/Raven.Client/Http/ClusterRequestExecutor.cs

```diff
@@ -16,6 +16,11 @@ public class ClusterRequestExecutor : RequestExecutor
 
         protected ClusterRequestExecutor(X509Certificate2 certificate, DocumentConventions conventions) : base(null, certificate, conventions)
         {
+            // Here we are explicitly ignoring trust issues in the case of ClusterRequestExecutor.
```

Comment:
I'm not really happy with this, to be honest, and we need to be sure that there aren't any usages of `ClusterRequestExecuter` outside of the Server Code.

That said, good enough for hte RC